pkgs/by-name/pi/pixi/package.nix

```diff
@@ -3,40 +3,37 @@
   stdenv,
   rustPlatform,
   fetchFromGitHub,
+  fetchpatch,
   pkg-config,
   libgit2,
   openssl,
   installShellFiles,
-  testers,
-  pixi,
+  buildPackages,
+  versionCheckHook,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "pixi";
-  version = "0.38.0";
+  version = "0.39.5";
 
   src = fetchFromGitHub {
     owner = "prefix-dev";
     repo = "pixi";
-    rev = "v${version}";
-    hash = "sha256-F15GDw6zolaa4IheKyJ9kdmdiLazUiDUhnUM8gH/hgk=";
+    tag = "v${version}";
+    hash = "sha256-Y4d8eMfsag2cNTaK8qnOGfi35kwwPrUy2y51EwVFrss=";
   };
 
-  postPatch = ''
-    # There are multiple `version-ranges` entries which is not supported by buildRustPackage.
-    cp -f ${./Cargo.lock} Cargo.lock
-  '';
-
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "async_zip-0.0.17" = "sha256-3k9rc4yHWhqsCUJ17K55F8aQoCKdVamrWAn6IDWo3Ss=";
-      "pubgrub-0.2.1" = "sha256-8TrOQ6fYJrYgFNuqiqnGztnHOqFIEDi2MFZEBA+oks4=";
-      "reqwest-middleware-0.3.3" = "sha256-KjyXB65a7SAfwmxokH2PQFFcJc6io0xuIBQ/yZELJzM=";
-      "tl-0.7.8" = "sha256-F06zVeSZA4adT6AzLzz1i9uxpI1b8P1h+05fFfjm3GQ=";
-      "uv-auth-0.0.1" = "sha256-xy/fgy3+YvSdfq5ngPVbAmRpYyJH27Cft5QxBwFQumU=";
-    };
-  };
+  # TODO: IMPORTANT! Remove this patch when the next version is released.
```

Comment:
This should conflict on the next update anyway, considering this commit is part of the project's main branch, so I don't think the "TODO: IMPORTANT!" comment prefix is necessary.

Reply:
I removed `IMPORTANT!` and keep the `TODO:` prefix

Reply:
because the `TODO:` prefix can be high light by vscode extension [TODO tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree)